projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts

```diff
@@ -193,6 +193,7 @@ describe('IgxGrid - Advanced Filtering #grid - ', () => {
 
             // Discard the new group and verify that the initial buttons are visible.
             GridFunctions.clickAdvancedFilteringExpressionCloseButton(fix);
+            fix.detectChanges();
```

Comment:
@igdmdimitrov can you check if those detectChanges() are ok?

Reply:
@Lipata , should be OK.